XRTK-Core/Packages/com.xrtk.core/EventDatum/Input/HandPanInputEventData.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+using XRTK.Interfaces.InputSystem;
+
+namespace XRTK.EventDatum.Input
+{
+    public class HandPanEventData : BaseInputEventData
+    {
+        public Vector2 PanPosition
```

Comment:
What does this even do?

Why not just put all the hand events into the same packet?

Reply:
I agree, shouldn't this be handled like "Pose"
```
IMixedRealityInputHandler<Hand>
```